packages/x-adapter-platform/src/types/responses/models/facet.model.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Facet model for the `platform` API.
+ *
+ * @public
+ */
+export interface PlatformFacet {
+  facet: string;
+  values: PlatformFilter[];
```

Comment:
The `values` can also be of type `PlatformHierarchicalFilter`, so the type here should add that possibility (either `PlatformFilter[] | PlatformHierarchicalFilter[]` or `(PlatformFilter | PlatformHierarchicalFilter)[]` I'm not sure the list can contain different kinds of filters).

Reply:
A `PlatformHierarchicalFilter` is a more complex `PlatformFilter` so, in the end, it is a `PlatformFilter`.